Modules/_ctypes/_ctypes.c

```diff
@@ -4793,6 +4793,15 @@ static PyMappingMethods Array_as_mapping = {
     Array_ass_subscript,
 };
 
+PyDoc_STRVAR(array_doc,
+"Abstract base class for arrays.\n"
+"\n"
+"The recommended way to create concrete array types is by multiplying any ctypes data type with\n"
```

Comment:
Can you make the line width within 80? This is common practice for code.

Reply:
No worries will change.